src/fsharp/symbols/SymbolHelpers.fs

```diff
@@ -1068,6 +1068,8 @@ module internal SymbolHelpers =
                 NicePrint.layoutILTypeRef denv finfo.ILTypeRef ^^
                 SepL.dot ^^
                 wordL (tagField finfo.FieldName) ^^
+                RightL.colon ^^
+                NicePrint.layoutTyconRef denv (tcrefOfAppTy g finfo.EnclosingType) ^^
```

Comment:
I'm not sure that it's correct. I suspect EnclosingType is not we should use here, doesn't it refer to DateTime for any field defined in DateTime struct?